web/client/components/widgets/builder/wizard/TableWizard.jsx

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright 2018, GeoSolutions Sas.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+const React = require('react');
+
+const { wizardHandlers } = require('../../../misc/wizard/enhancers');
+
+const TableOptions = require('./table/TableOptions');
+const WidgetOptions = require('./common/WidgetOptions');
+const isChartOptionsValid = (options = {}) => options.aggregateFunction && options.aggregationAttribute && options.groupByAttributes;
+
+const Wizard = wizardHandlers(require('../../../misc/wizard/WizardContainer'));
+
+const { compose, lifecycle } = require('recompose');
+
+const enhanceWizard = compose(lifecycle({
```

Comment:
How do you enhance the wizard?

Reply:
I would call this something that recalls validation

Reply:
Good, going to change it